### ISSUE TITLE
when filtering is applied we should be able to hover over the information icon and see what conditions are being applied

### ISSUE THREAD

**Issue by Luke1982** [tsolucio/corebos#1034]

**Comment by Luke1982** [comment]
@joebordes Could you elaborate on which information icon you mean?

**Comment by joebordes** [comment]
![image](https://user-images.githubusercontent.com/1237191/114302459-66e01e80-9ac9-11eb-826e-bce6b5dedbd0.png)

"Filter Applied" green icon. This is just a "nice to have" as the filter conditions can be seen above.

**Comment by Luke1982** [comment]
Hmm well I think it is more than that. I think it could really speed up your workflow to be able to see those conditions at a glace.

**Comment by joebordes** [comment]
I agree. I just tried and the column header search/filter is totally invisible. In the image above I applied a search in the coreBOS ad-hoc filter, so you can see the condition there, but if you click on the filter icon in the column header and apply the filter there, it works but you lose all context of what the condition is. It isn't even in the column filter if you click on it again.
